CHANGELOG.next.asciidoc

```diff
@@ -967,6 +967,18 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Affecting all Beats*
 
+- Update to Golang 1.15.6 requires users to add `x509ignoreCN=0` to the `GODEBUG` environment variable if
+they are using a X.509 certificates without Subject Alternative Names in the following use cases:
+ * running Beats/Agent on Windows
+ * compiling Beats/Agent and then running the binary
```

Comment:
@ph There is an issue to fix this in Agent, right?

This can affect certificate validation in general. If we talk to ES/Kibana/Logstash/Kafka/... via TLS and the certificate has Common Name set, then we have a problem.